src/SamplesApp/SamplesApp.UITests/UnoSamples_Tests.Keyboard.cs

```diff
@@ -446,6 +446,8 @@ public void Keyboard_DismissTesting()
 			// We only validate that the bottom of the screen is the same (so the keyboard is no longer visible).
 			// This is to avoid content offset if the status bar was opened by the keyboard or the message box.
 			ImageAssert.AreEqual(initial, final, new Rectangle(0, -100, int.MaxValue, 100));
+
+			_app.Back();
```

Comment:
The test run for this PR fails consistently on Wasm. Why do need to add a back here?

Reply:
I made a mistake, removed and all checks passes!